lib/twitch.rb

```diff
@@ -33,7 +33,7 @@ def self.followed_ids(login)
 
     class << self
       def get(login)
-        route(login).get(Twitch.headers)
+        route(login.twitch_id).get(Twitch.headers)
```

Comment:
Should I change that to only pass id as well?  That one was passing a user object in from the beginning, and I didn't change how the function was called.  Previously it would implicitly .to_s the user object.  The only place I could see it called was from twitch_user.rb on line 10.